### PR TITLE
APP-116 Fix cassandra data demo

### PR DESCRIPTION
**Pull request by Dubland**
/cc @rasendubi @forGGe

### CODE REVIEW

**Comment by ohamov** [comment]
LGTM

**Comment by rasendubi** [comment]
LGTM

File: cassandrademo/source/c/CMakeLists.txt [hunk 1]
```diff
@@ -15,25 +15,23 @@
 #
 
 cmake_minimum_required(VERSION 2.8.8)
+project(C-SDK-sample C)
 
-project (C-SDK-sample)
-enable_language(C)
+find_package(OpenSSL REQUIRED)
+find_package(Threads REQUIRED)
```
Comment (forGGe) [inline cassandrademo/source/c/CMakeLists.txt:21]: Not sure that threads library is required. Please, check.

File: cassandrademo/source/c/build.sh [hunk 2]
```diff
@@ -31,59 +31,24 @@ fi
...
-function clean {
-    rm -rf "$KAA_LIB_PATH/$BUILD_DIR"
+clean() {
     rm -rf "$PROJECT_HOME/$BUILD_DIR"
 }
 
-function run {
+run() {
     cd "$PROJECT_HOME/$BUILD_DIR"
     sudo ./$APP_NAME
```
Comment (forGGe) [inline cassandrademo/source/c/build.sh]: Not sure that `sudo` is required. Please, check.

**Comment by forGGe** [comment]
Reviewed.

File: cassandrademo/source/c/CMakeLists.txt (hunk 1 above)
Comment (Dubland) [inline cassandrademo/source/c/CMakeLists.txt:21]: This error occurs without thread:

```
/usr/lib/gcc/x86_64-linux-gnu/5/../../../../lib/libwiringPi.so: undefined reference to `pthread_create'
/usr/lib/gcc/x86_64-linux-gnu/5/../../../../lib/libwiringPi.so: undefined reference to `pthread_cancel'
/usr/lib/gcc/x86_64-linux-gnu/5/../../../../lib/libwiringPi.so: undefined reference to `pthread_join'
```

**Comment by forGGe** [comment]
Looks good, merging.

**Pull request merged**